src/DynamoCore/DynamoCore.csproj

```diff
@@ -21,9 +21,11 @@
     <Reference Include="System.Configuration" />
   </ItemGroup>
   <ItemGroup>
-	<PackageReference Include="DynamoVisualProgramming.LibG_227_0_0" Version="2.13.0.8745" GeneratePathProperty="true">
-		<ExcludeAssets>all</ExcludeAssets>
-	</PackageReference>
+    <PackageReference Include="Lib.Harmony" Version="2.2.1" ExcludeAssets="runtime" Condition="'$(Configuration)' != 'Debug'" />
+    <PackageReference Include="Lib.Harmony" Version="2.2.1" ExcludeAssets="none" Condition="'$(Configuration)' == 'Debug'" />
+	  <PackageReference Include="DynamoVisualProgramming.LibG_227_0_0" Version="2.13.0.8745" GeneratePathProperty="true">
```

Comment:
Debug builds will copy the harmony dll to the bin folder.
Release builds will not copy it.